casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -230,18 +230,18 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
     "(!C) CX"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(S0 | C_, C_, Nil),
     "(4!) (4!)" -> VOLATILE_EVENT             -> merges(F_ | S0, F_ | S0, Nil),
     "(4!) (C!)" -> VOLATILE_EVENT             -> merges(F_ | S0, C_ | S0, Nil),
-    "(4!) 4!"   -> VOLATILE_EVENT             -> failsRandomly(F_ | S0, F_, S0),
-    "(4!) C!"   -> VOLATILE_EVENT             -> failsRandomly(F_ | S0, C_, S0),
+    "(4!) 4!"   -> VOLATILE_EVENT             -> merges(F1 | S1, F_, S0),
+    "(4!) C!"   -> VOLATILE_EVENT             -> merges(F1 | S1, C_, S0),
     "(C!) C!"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(C_ | S0, C_, S0),
-    "4! (4!)"   -> VOLATILE_EVENT             -> failsRandomly(F_, F_ | S0, S0),
+    "4! (4!)"   -> VOLATILE_EVENT             -> merges(F_, F1 | S1, S0),
     "4! (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(F_, C_ | S0, S0),
     "4X (4!)"   -> VOLATILE_EVENT             -> merges(F_, F_ | S0, Nil),
     "4X (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(F_, C_ | S0, Nil),
     "C! (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(C_, C_ | S0, S0),
     "CX (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflictsForNow(C_, C_ | S0, Nil),
-    "(!4) 4!"   -> VOLATILE_EVENT             -> failsRandomly(S0 | F_, F_, S0),
-    "(!4) !C"   -> VOLATILE_EVENT             -> failsRandomly(S0 | F_, S0, C_),
-    "(!4) C!"   -> VOLATILE_EVENT             -> failsRandomly(S0 | F_, C_, S0),
+    "(!4) 4!"   -> VOLATILE_EVENT             -> merges(S1 | F1, F_, S0),
```

Comment:
Nit: Ideally, we'd use lower indices first and this would be `merges(S0 | F0, F_, S1)`.